src/services/product.service.ts

```
@@ -93,25 +154,252 @@ export const productService = {
+    const anyPayload = payload as any;
+    if (anyPayload.title) form.append("title", String(anyPayload.title));
+    if (anyPayload.categoryId)
+      form.append("categoryId", String(anyPayload.categoryId));
+    if (anyPayload.price !== undefined)
+      form.append("price", String(anyPayload.price));
+    if (anyPayload.isNegotiable !== undefined)
+      form.append("isNegotiable", String(Boolean(anyPayload.isNegotiable)));
+    if (anyPayload.condition)
+      form.append("condition", toBackendCondition(anyPayload.condition)!);
+    if (Array.isArray(anyPayload.images)) {
+      anyPayload.images.forEach((file: File) => form.append("images", file));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)